README.md

```diff
@@ -163,6 +163,98 @@ async function keyProtectSdkExample() {
 keyProtectSdkExample();
 ```
 
+### Example of wrap/unwrap/delete key using key alias-extensions in SDK
+
```

Comment:
Suggestion: remove these updates in README, instead have a separate PR to update the examples in README with all/most of the remaining functions